Cargo.toml

```diff
@@ -15,4 +15,7 @@ rand = "0.7.3"
 [dependencies.bulletproofs]
 git = "https://github.com/dusk-network/bulletproofs"
 branch = "develop"
-features = ["yoloproofs"]
\ No newline at end of file
+features = ["yoloproofs"]
+
+[profile.dev]
+opt-level = 3
```

Comment:
I don't understand this part. The dev/check build gets too slow; if we need to run tests, we could do something like `$ cargo test --release`?

Reply:
Ye, I had the Cfg test option set, will remove it! 
Thanks mate!